catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.hbs

```diff
@@ -103,6 +109,101 @@
                 <input class="form-control" id="usng" name="usng" type="text" placeholder=""/>
             </div>
 
+            <div id="utmdivEasting" class="input-group input-group-sm" style="display: none;">
+                <span class="input-group-addon utm-label">Easting&nbsp;</span>
```

Comment:
Should these labels (Easting, Northing, etc) really have non-breaking spaces in them?